### PR TITLE
Set combinations attributes closed by default if combinations > 0

### LINKED ISSUE #16100: Collapse right column if combinations have already been generated
When no combinations have been generated yet, attributes should be all opened and values displayed (currently they are not all opened)

capture d'écran_2103

If combinations are already generated, attributes should be collapsed and values shouldn't be displayed.

capture d'écran_2104

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Combinations attributes are taking too much places if they have been already generated
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #16100.
| How to test?      | Add new product with combination, go on combinations pan, see that attributes are openned, generate combinations, refresh the page and they should be closed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23180)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_combinations.html.twig [hunk 1]
```diff
@@ -127,14 +127,14 @@
     {% for attribute_group in attribute_groups %}
       <div class="attribute-group">
         <a
-          class="attribute-group-name {% if loop.index > 3 %} collapsed {% endif %}"
+          class="attribute-group-name {% if loop.index > 3 or has_combinations %} collapsed {% endif %}"
           data-toggle="collapse"
           href="#attribute-group-{{ attribute_group.id }}"
-          aria-expanded="{% if loop.index <= 3 %}true{% else %}false{% endif %}"
+          aria-expanded="{% if loop.index <= 3 or has_combinations %}true{% else %}false{% endif %}"
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_combinations.html.twig]:
```suggestion
          aria-expanded="{% if loop.index <= 3 or not has_combinations %}true{% else %}false{% endif %}"
```

**Review by Progi1984 — COMMENTED** [review]
Small feedback

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by SD1982** [comment]
LGTM !!
**Before PR**
https://www.awesomescreenshot.com/video/2705000?key=2e4cf0494e3088264e995f5d61aa34b3
**After PR**
https://www.awesomescreenshot.com/video/2705017?key=5a5020ac22e363ff67788d93f755a301
Thanks @NeOMakinG !!

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @SD1982

**Pull request merged**